src/styles/styles.scss

```diff
@@ -196,18 +189,18 @@
 				&:active {
 					box-shadow: 0 0 0 2px var(--color-main-text);
 				}
+			}
 
-				&--tabbed {
-					box-shadow: 0 0 0 2px var(--color-main-text);
-					background-color: var(--color-primary-light-hover);
+			&--tabbed:not(.primary) {
```

Comment:
Was causing issue with compilation, simply rearranged

Reply:
Now it's also affecting a-href links and seems to not look good anymore (it becomes square)?

Also the colors seem to fall apart
![Bildschirmfoto vom 2022-06-02 12-33-45](https://user-images.githubusercontent.com/213943/171611382-5ce1b67a-9212-49aa-902f-e289f87791fb.png)

Reply:
Issue already exist outside of this PR.
Fix here: https://github.com/nextcloud/server/pull/32713